src/webhooks/travis.js

```diff
@@ -0,0 +1,39 @@
+const fetch = require('node-fetch');
+const crypto = require('crypto');
+
+const config = require('../config');
+
+const deploy = require('../worker/deploy');
+
+module.exports = (req, res) => {
+  const options = {};
+  try {
+    const metadata = JSON.parse(req.body.payload);
+    options.branchName = metadata.branch;
+    options.isPullRequest = metadata.pull_request;
+
+    options.ownerName = metadata.repository.owner_name;
+    options.repoName = metadata.repository.name;
+    options.commitHash = metadata.commit;
+  } catch (e) {
+    return res.status(400).send(e.stack);
+  }
+
+  if (config.manager.trustWebhooks) {
```

Comment:
Maybe renaming this config `blindlyTrustWebhooks` would convey even more the fact that it's a *very* bad idea?